app/services/event_generator/geoblacklight_event_generator.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+class EventGenerator
+  class GeoblacklightEventGenerator
```

Comment:
These are really great. Maybe a shared spec for what the expected event generator interface is?

Reply:
Added shared spec.